src/pages/export.html

```diff
@@ -98,16 +96,14 @@ <h1>
 										<button type="button" class="btn btn-link dropdown-toggle" uib-dropdown-toggle gdb-tooltip="{{'export.selected.graphs.tooltip' | translate}}" ng-disabled="!hasMultipleSelected()">
 											<span class="icon-export"></span></button>
 										<ul class="dropdown-menu" role="menu">
-                                            <li ng-repeat="format in exportFormats" class="export-repo-format-{{format.name}}"
-                                                ng-if="format.name === 'JSON-LD';"
-                                                ng-click="openJSONLDExportSettingsForSelectedGraphs(format);">
-                                                <span class="dropdown-item">{{format.name}}</span>
+                                            <li ng-repeat="format in exportFormats">
+                                                <span ng-click="openJSONLDExportSettingsForSelectedGraphs(format)"
```

Comment:
Do you mind replacing these spans with links in order the html to be semantically valid?

Reply:
Sure, I'll replace all of them.